command/vault/secrets_pull.go

```diff
@@ -95,14 +95,14 @@ func SecretsPull(c *cli.Context) error {
 			if err != nil {
```

Comment:
this is a breaking, unrelated change to your PR title, please open a new PR for discussion on this